ksqldb-rest-app/src/main/java/io/confluent/ksql/api/auth/JaasAuthProvider.java

```diff
@@ -51,7 +50,7 @@
   private final String contextName;
 
   public JaasAuthProvider(final Server server, final KsqlRestConfig config) {
-    this(server, config, LoginContext::new);
+    this(server, config, () -> new JAASLoginService());
```

Comment:
IntelliJ pointed out that this can be `JAASLoginService::new` instead if you like.  :shrug:

Reply:
ah yes! I had that at first, and then added params, and then removed params and the auto-refactoring got me 😉  I'll make that change